packages/types/src/protocol.ts

```diff
@@ -3,7 +3,7 @@ import type { SpecFile } from '.'
 // TODO(protocol): This is basic for now but will evolve as we progress with the protocol work
 
 export interface AppCaptureProtocolInterface {
-  connectToBrowser (options: { target: string, host: string, port: number }): void
+  connectToBrowser (options: { target: string, host: string, port: number }): Promise<void>
```

Comment:
ATM this isn't returning a promise, it's just async unless you apply my above suggestion